src/shell/store/content.js

```diff
@@ -977,7 +977,7 @@ export function fetchItemPublishings() {
   return (dispatch) => {
     return dispatch({
       type: "FETCH_RESOURCE",
-      uri: `${CONFIG.API_INSTANCE}/content/items/publishings?limit=100000`,
+      uri: `${CONFIG.API_INSTANCE}/content/items/publishings?limit=20000`,
```

Comment:
Can we add a comment to clearly delineate this is the upper bound.

How as this upper bound determined?